README.md

```diff
@@ -1,28 +1,28 @@
 # stoa
 API server for Agora
 
+# Build instructions
+Install the Node.js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
 
-
-## Build instructions
-
-# Install the node.Js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
-
-
-# Building on Ubuntu
-# Ubuntu is already nodejs installed.
+## Building on Ubuntu
+Ubuntu is already Node.js installed.
```

Comment:
The correct sentence would be "Ubuntu already has Node.js installed". But is that true, or just on the CI machines ?

Reply:
It's true.
https://github.com/nodesource/distributions/blob/master/README.md

Reply:
Oh, should I say it's included?

Reply:
Delete this sentence.